qualtran/bloqs/multiplexers/apply_gate_to_lth_target.py

```diff
@@ -104,6 +106,10 @@ def nth_operation(  # type: ignore[override]
         target_idx = int(np.ravel_multi_index(selection_idx, selection_shape))
         return self.nth_gate(*selection_idx).on(target[target_idx]).controlled_by(control)
 
+    def nth_operation_callgraph(self, **selection_regs_name_to_val) -> Set['BloqCountT']:
+        selection_idx = tuple(selection_regs_name_to_val[reg.name] for reg in self.selection_regs)
+        return {(_cirq_gate_to_bloq(self.nth_gate(*selection_idx)), 1)}
```

Comment:
The `self.target_gate` and `self.nth_gate` should be controlled gates here and below.

Reply:
I think this would be a good opportunity to revisit these gates, use the new controlled infrastructure, create a bloq example and fix whatever is failing.